lib/fluent/plugin/in_secure_forward.rb

```diff
@@ -91,9 +90,30 @@ def initialize
       define_method("router") { Fluent::Engine }
     end
 
+    def replace_hostname_placeholder(conf)
+      hostname = @hostname ? @hostname : Socket.gethostname
+
+      placeholders = ['__HOSTNAME__', '${hostname}']
+
+
+      check_element = ->(placeholders, c) {
+        c.keys.each do |k|
+          v = c.fetch(k, nil)
+          if v and placeholders.include? v
```

Comment:
`placeholders.include? v` works only when `v == "${hostname}"` or `v == "__HOSTNAME__"`. But it should work when string values includes `${hostname}` or `__HOSTNAME__` (and replace these placeholders with hostname string).